### PR TITLE
cni-plugin.md: document "mtu" and "trunk"

### PR DESCRIPTION
**Pull request by jsquyres**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Trivial update to cni-plugin.md documentation file.

**Which issue(s) this PR fixes**:

Does not fix any issue.

**Special notes for your reviewer**:

I am a first time contributor.  I only found the process stuff (e.g., formatting the PR description) from the bot comments after I initially filed this PR.  Additionally, there's no `.github` directory in this repo with a PR template, etc.  I had to get this template from https://github.com/kubernetes/kubernetes.

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```
**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

File: docs/cni-plugin.md [hunk 1]
```diff
@@ -17,7 +17,9 @@ Please note that Open vSwitch must be installed and running on the host.
     "name": "mynet",
     "type": "ovs",
     "bridge": "mynet0",
-    "vlan": 100
+    "vlan": 100,
+    "mtu": 9000,
+    "trunk": [ { "id" : 42 }, { "minID" : 1000, "maxID" : 1010 } ]
```
Comment (phoracek) [inline docs/cni-plugin.md]: Is it valid to have both "vlan" and "trunk" set at the same time?

Comment (jsquyres) [reply docs/cni-plugin.md]: Good point; probably not.  I was just putting all the fields in there, which is misleading at best.

I can split it into 2 examples. 👍

**Comment by phoracek** [comment]
Sorry about the bots. We just recently integrated prow into this repo and it has some loose ends.

**Comment by jsquyres** [comment]
> Sorry about the bots. We just recently integrated prow into this repo and it has some loose ends.

No worries!  They were actually quite helpful in understanding the process.

**Pull request merged**